internal/cmd/gocdk/deploy.go

```diff
@@ -43,18 +44,19 @@ By default, a new Docker image is built and deployed; use --image to skip the
 build and use an existing tagged image instead.`,
 		Args: cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
-			return deploy(ctx, pctx, args[0], dockerImage, apply)
+			return deploy(ctx, pctx, args[0], dockerImage, apply, verbose)
 		},
 	}
 	deployCmd.Flags().StringVar(&dockerImage, "image", "", "Docker image to deploy in the form `[name][:tag]`; name defaults to image name from Dockerfile, tag defaults to latest; empty string builds a new image")
-	deployCmd.Flags().BoolVar(&apply, "apply", true, "whether to run `biome apply` before deploying")
+	deployCmd.Flags().BoolVar(&apply, "apply", true, "whether to run \"biome apply\" before deploying")
+	deployCmd.Flags().BoolVar(&verbose, "verbose", false, "true to print all output from Terraform during \"biome apply\"")
 	rootCmd.AddCommand(deployCmd)
 }
 
 // deploy implements the "deploy" command.
 //
 // If dockerImage is not provided, it does a build for a generated tag.
-func deploy(ctx context.Context, pctx *processContext, biome, dockerImage string, apply bool) error {
+func deploy(ctx context.Context, pctx *processContext, biome, dockerImage string, apply, verbose bool) error {
```

Comment:
Both `apply` and `verbose` could be put into an options struct since they are not required.

Reply:
Done. I re-used `biomeApplyOptions` in case we want to add flag for the other fields.